.circleci/config.yml

```diff
@@ -144,6 +151,20 @@ jobs:
             - test/copyimg/checkseccomp/checkseccomp
             - test/copyimg/copyimg
 
+  bundle:
+    executor: docker
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: bundle
+          command: make bundle
+      - persist_to_workspace:
+          root: .
+          paths:
+            - bundle/*.tar.gz
+
   clang-format:
     executor: docker
```

Comment:
No change required for this PR, but could we replace docker with podman ?

Reply:
Yeah I thought about using podman there as well. It would mean some slight changes to the CI, but in general it should work (even rootless). I will put it on my TBD list. :)